server/test/reypayments.test.js

```diff
@@ -0,0 +1,206 @@
+      .patch('/api/v1/loans/2')
+      .set('x-access-token', token)
+      .send({
+        status: 'approve'
+      })
+      .end((err, res) => {
+        res.should.have.status(200);
+        done();
+      });
+  });
+  it(`should return 200 for repayments`, done => {
+    chai
+      .request(app)
+      .post('/api/v1/loans/2/repayments')
+      .set('x-access-token', token)
+      .end((err, res) => {
+        res.should.have.status(201);
+        res.body.status.should.equal('Success');
+        res.body.message.should.equal('Repayment transaction was successfull');
+        done();
+      });
+  });
+  it(`should return 400 for applying why there is outstanding loan`, done => {
+    chai
+      .request(app)
+      .post('/api/v1/loans')
+      .set('x-access-token', token)
+      .send({
+        amount: 100000,
+        tenor: 4
+      })
+      .end((err, res) => {
+        res.should.have.status(400);
+        res.body.error.should.equal("You cannot apply. You still have outstanding");
+        done();
+      });
+  });
+  it(`should return 200 for repayments`, done => {
+    chai
+      .request(app)
+      .post('/api/v1/loans/2/repayments')
+      .set('x-access-token', token)
+      .end((err, res) => {
+        res.should.have.status(201);
+        res.body.status.should.equal('Success');
+        res.body.message.should.equal('Repayment transaction was successfull');
+        done();
+      });
+  });
+  it(`should return 200 for repayments`, done => {
+    chai
+      .request(app)
+      .post('/api/v1/loans/2/repayments')
+      .set('x-access-token', token)
+      .end((err, res) => {
+        res.should.have.status(201);
+        res.body.status.should.equal('Success');
+        res.body.message.should.equal('Repayment transaction was successfull');
+        done();
+      });
+  });
+  it(`should return 200 for repayments`, done => {
+    chai
+      .request(app)
+      .post('/api/v1/loans/2/repayments')
+      .set('x-access-token', token)
+      .end((err, res) => {
+        res.should.have.status(201);
+        res.body.status.should.equal('Success');
+        res.body.message.should.equal('Repayment transaction was successfull');
+        done();
+      });
+  });
+  it(`should return 400 for repayments`, done => {
+    chai
+      .request(app)
+      .post('/api/v1/loans/2/repayments')
+      .set('x-access-token', token)
+      .end((err, res) => {
+        res.body.status.should.equal(400);
+        res.body.error.should.equal('Loan with the provided id has been fully paid');
+        done();
+      });
+  });
+  it(`should return 200 for repayments`, done => {
+    chai
+      .request(app)
+      .get('/api/v1/loans/2/repayments')
+      .set('x-access-token', token)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.status.should.equal('Success');
+        res.body.message.should.equal('Repayments retrieve successfully');
+        done();
+      });
+  });
+})
```

Comment:
please add an extra line to follow the Eslint standard

Reply:
Noted